packages/astro/src/cli/add/index.ts

```diff
@@ -83,20 +83,16 @@ export default async function seed() {
 `,
 	CLOUDFLARE_WRANGLER_CONFIG: (name: string) => `\
 {
-	"main": "dist/_worker.js/index.js",
+	"compatibility_date": "2025-05-21",
```

Comment:
I think here it makes sense to keep the dynamic generation:

> When you start your project, you should always set compatibility_date to the current date.

https://developers.cloudflare.com/workers/configuration/compatibility-dates/

Reply:
Internally we have a helper that returns the latest version supported by the installed version of miniflare. We could add an export from the Vite plugin.